subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -60,7 +60,7 @@ Users can still opt in to have this unique identifier part of the produced metad
 [[jcenter_deprecation]]
 === The `jcenter()` convenience method is now deprecated
 
-JFrog recently link:https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter[announced] the decommission of JCenter.
+JFrog link:https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter[announced] the decommission of JCenter.
 Gradle emits a deprecation warning when `jcenter()` is used in the repository configuration.
 Consider <<declaring_repositories.adoc#declaring-repositories,using>> `mavenCentral()`, `google()` or a private `maven` repository instead.
```

Comment:
I think we should replace this sentence with a link to our blog post on the topic.

Reply:
That's a good point, did not think of it. Will do